redcap_bridge/utils.py

```diff
@@ -277,6 +287,8 @@ def parse_choices(choice_str, annotation_str):
             else:
                 warnings.warn(f'Could not determine default choice for {annotation_str}')
 
-    return list(choice_labels), default_choice_label
+    choice_labels = [re.sub(r'{|}', '', label) for label in choice_labels]
```

Comment:
This definitely requires a comment if you want to be able to understand later on why you did this...